en/dev_airframes/adding_a_new_frame.md

```diff
@@ -2,7 +2,6 @@
 
 PX4 uses canned airframe configurations as starting point for airframes.
 The configurations are defined in [config files](#config-file) that are stored in the [ROMFS/px4fmu_common/init.d](https://github.com/PX4/PX4-Autopilot/tree/main/ROMFS/px4fmu_common/init.d) folder.
-The config files reference [mixer files](#mixer-file) that describe the physical configuration of the system, and which are stored in the [ROMFS/px4fmu_common/mixers](https://github.com/PX4/PX4-Autopilot/tree/main/ROMFS/px4fmu_common/mixers) folder.
```

Comment:
@bkueng This is minimal changes to adding a new frame to explain what you do. 

I'd have gone further except the wing-wing airframe does not use control allocation. To make an example of control allocation in place I'd have to have changed to another airframe, which would have made the rest of the mixer docs "off".

Anyway, if wing wing gets CA params then I would update this. Otherwise when we move to v1.14 this can all be deleted and we'll move to a different frame.

Reply:
Right now it just points to the control allocation doc for info about how things might work. I "think" that is enough.